src/Sarif.UnitTests/Visitors/InsertOptionalDataVisitorTests.cs

```diff
@@ -3,138 +3,111 @@
 
 using System;
 using System.Collections.Generic;
-using System.Globalization;
 using System.IO;
-using System.Text;
 using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif.TestUtilities;
 using Microsoft.CodeAnalysis.Sarif.Writers;
 using Newtonsoft.Json;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Microsoft.CodeAnalysis.Sarif.Visitors
 {
-    public class InsertOptionalDataVisitorTests : FileDiffingTests
+
+    public class InsertOptionalDataVisitorTests : FileDiffingTests, IClassFixture<InsertOptionalDataVisitorTests.InsertOptionalDataVisitorTestsFixture>
```

Comment:
>InsertOptionalDataVisitorTests [](start = 17, length = 30)

converted this test suite to current pattern. suggest just looking at the new file rather than the diffs. will be most informative.